[2022-11-17 23:43:31,554] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_dag.call_hello 2022-11-17T23:43:28.189304+00:00 [queued]>
[2022-11-17 23:43:31,610] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_dag.call_hello 2022-11-17T23:43:28.189304+00:00 [queued]>
[2022-11-17 23:43:31,612] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-11-17 23:43:31,614] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-11-17 23:43:31,615] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-11-17 23:43:31,647] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): call_hello> on 2022-11-17T23:43:28.189304+00:00
[2022-11-17 23:43:31,681] {standard_task_runner.py:52} INFO - Started process 99 to run task
[2022-11-17 23:43:31,701] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_dag', 'call_hello', '2022-11-17T23:43:28.189304+00:00', '--job-id', '349', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/index.py', '--cfg-path', '/tmp/tmplnyh74hy', '--error-file', '/tmp/tmp_r6wimnh']
[2022-11-17 23:43:31,710] {standard_task_runner.py:77} INFO - Job 349: Subtask call_hello
[2022-11-17 23:43:32,055] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_dag.call_hello 2022-11-17T23:43:28.189304+00:00 [running]> on host 1d9aa7be56f4
[2022-11-17 23:43:32,255] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_dag
AIRFLOW_CTX_TASK_ID=call_hello
AIRFLOW_CTX_EXECUTION_DATE=2022-11-17T23:43:28.189304+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-17T23:43:28.189304+00:00
[2022-11-17 23:43:32,266] {python.py:118} INFO - Done. Returned value was: hello everyone
[2022-11-17 23:43:32,398] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=hello_dag, task_id=call_hello, execution_date=20221117T234328, start_date=20221117T234331, end_date=20221117T234332
[2022-11-17 23:43:32,523] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-11-17 23:43:32,577] {local_task_job.py:146} INFO - Task exited with return code 0
